src/sort.rs

```diff
@@ -1,6 +1,6 @@
 use crate::flags::{DirGrouping, Flags, SortColumn, SortOrder};
 use crate::meta::Meta;
-use human_sort::compare;
+use crate::versionsort::compare;
```

Comment:
I think you didn't meant to simply remove human-sort, but this is what you are doing here. 

Did you forget to implement some cli / config switch ?

Reply:
I do mean to replace the sorting library. The new code is now its own separate crate and the PR now reflects that.